views/github-create-branch.hbs

```diff
@@ -21,15 +21,61 @@
 <input type="hidden" id="_csrf" name="_csrf" value="{{csrfToken}}" />
 <input type="hidden" id="jiraHost" name="jiraHost" value="{{jiraHost}}" />
 
-{{> navigation hideBackButton=true }}
+{{> navigation }}
 
 <section class="gitHubCreateBranch">
   <div class="headerImage">
     <img src="/public/assets/jira-and-github.png" alt="Jira and GitHub logos" />
   </div>
   <div class="gitHubCreateBranch__title">
-    Create a Branch
+    <div class="gitHubCreateBranch__header">Create a Branch</div>
+    <div class="gitHubCreateBranch__subHeader">
+      Creating a branch for
+      <!--TODO: Need to add icons based on the type of issue(story/task/bug) -->
+      <a href="#">KEY-1</a>
+    </div>
   </div>
+
+  <div class="gitHubCreateBranch__optionContainer">
```

Comment:
Don't we want to hide this unless the customer has both?

Reply:
Actually, if we have to go through auth before this do we even need this? This is similar to the connect an org page in the sense that we've already done the auth handshake, then we render what the user has access to for that server/app/org.

Reply:
Yeah, feel free to drop this section for now, we are still figuring out the ins and outs when it comes to authentication.
Has its own story to deal with it.

Reply:
This has been removed for this task.